charts/otel-cloud-stack/values.yaml

```diff
@@ -0,0 +1,476 @@
+extraEnvs: []
+
+## Auto-Instrumentation resource to be installed in the cluster
+## Can be used by setting the following:
+##  Java: instrumentation.opentelemetry.io/inject-java: "true"
+##  NodeJS: instrumentation.opentelemetry.io/inject-nodejs: "true"
+##  Python: instrumentation.opentelemetry.io/inject-python: "true"
+##  DotNet: instrumentation.opentelemetry.io/inject-dotnet: "true"
+##  OpenTelemetry SDK environment variables only: instrumentation.opentelemetry.io/inject-sdk: "true"
+autoinstrumentation:
+  enabled: false
+  ## The collector name to send traces to
+  collectorTarget: traces
+  propagators:
+    - tracecontext
+    - baggage
+    - b3
+
+  ## Sampler defines the OTEL sampler behavior to be used. Example:
+  ##
+  ## sampler:
+  ##   type: parentbased_traceidratio
+  ##   argument: "0.25"
+  ##
+  sampler:
+    ## The value can be for instance parentbased_always_on, parentbased_always_off, parentbased_traceidratio...
+    type: parentbased_traceidratio
+    ## The value depends on the sampler type.
+    ## For instance for parentbased_traceidratio sampler type it is a number in range [0..1] e.g. 0.25.
+    argument: "0.25"
+
+  ## A list of corev1.EnvVars
+  env: []
+
+  ## https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/overview.md#resources
+  resource: {}
+
+daemonCollector:
+  name: daemon
+  image: otel/opentelemetry-collector-contrib:0.83.0
+  enabled: true
+  mode: daemonset
+  mountHostFS: true
+  env:
+    - name: LS_TOKEN
+      valueFrom:
+        secretKeyRef:
+          key: LS_TOKEN
+          name: otel-collector-secret
+  resources:
+    limits:
+      cpu: 250m
+      memory: 500Mi
+    requests:
+      cpu: 250m
+      memory: 128Mi
+  scrape_configs_file: "daemon_scrape_configs.yaml"
+  config:
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:4317
+      kubeletstats:
+        collection_interval: "15s"
+        auth_type: "serviceAccount"
+        insecure_skip_verify: true
+        # For this scrape to work, the RBAC must have `nodes/stats` GET access.
+        endpoint: "https://${env:K8S_NODE_NAME}:10250"
+        extra_metadata_labels:
+          - container.id
+          - k8s.volume.type
+        metric_groups:
+          - node
+          - pod
+          - volume
+          - container
+        k8s_api_config:
+          auth_type: serviceAccount
+      hostmetrics:
```

Comment:
nit: option to make hostmetrics optional? (if you can't get access to host filesystem)

Reply:
ugh so its pretty hard to do this because this is in the values.yaml not in the templates. A customer could manually disable this by overriding the pipeline?